### PR TITLE
refactor: changes to use wallet-lib v1.1.0

### PR DESCRIPTION
**Pull request by pedroferreira1**
### Acceptance Criteria
- Bump lib version to the latest (v1.1.0)
  - The previous lib version was really old and many lib dependencies have changed. Because of that, the `package-lock.json` update was not easy, then I decided to also fix the versions of all dependencies in the `package.json` with the ones from the `package-lock.json` and run a full update in the `package-lock.json` file.
- Update code for the new lib methods

### Security Checklist
- [x] Make sure you do not include new dependencies in the project unless strictly necessary and do not include dev-dependencies as production ones. More dependencies increase the possibility of one of them being hijacked and affecting us.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -118,6 +116,7 @@ class Root extends React.Component {
               <NavigationRoute exact path="/statistics" component={Dashboard} />
               <NavigationRoute exact path="/token_detail/:tokenUID" component={TokenDetail} />
               <NavigationRoute exact path="/address/:address" component={AddressDetail} />
+              <NavigationRoute exact path="/nano_contract/detail/:nc_id" component={NanoContractDetail} />
```
Comment (r4mmer) [inline src/App.js:119]: Is this part of this PR?

Comment (pedroferreira1) [reply src/App.js]: Yes, this belongs to the other PR (the nano integration PR), my bad.

Removed at https://github.com/HathorNetwork/hathor-explorer/pull/252/commits/9b6888b5228997e492b5bb854eb0eef51c56e8a5

File: src/screens/DashboardTx.js [hunk 2]
```diff
@@ -26,11 +27,11 @@ class DashboardTx extends React.Component {
 
   componentDidMount = () => {
     this.getInitialData();
-    hathorLib.WebSocketHandler.on('network', this.handleWebsocket);
```
Comment (tuliomir) [inline src/screens/DashboardTx.js:29]: question: This change seems reasonable to me, but it wasn't informed on the acceptance criteria and it doesn't seem directly related to the lib upgrade.

Is this related to the context of this PR?

File: package.json [hunk 3]
```diff
@@ -3,33 +3,33 @@
...
+    "d3-selection": "1.4.2",
+    "d3-zoom": "1.8.3",
+    "font-awesome": "4.7.0",
+    "jquery": "3.7.0",
+    "lodash": "4.17.21",
+    "npm-run-all": "4.1.5",
+    "popper.js": "1.16.1",
+    "prop-types": "15.8.1",
+    "qrcode.react": "0.8.0",
+    "react": "16.14.0",
```
[the remark this answers is not in the document]
Comment (pedroferreira1) [reply package.json:18]: We should have a project to do dependencies upgrades for the explorer, just like we are having for the wallets. Even though is less critical, it's also important.

File: src/screens/DashboardTx.js (hunk 2 above)
Comment (pedroferreira1) [reply src/screens/DashboardTx.js:29]: It is part of the same upgrade because we no longer have `WebSocketHandler` in the lib. However, we do have this class in the explorer itself, so I just changed it to use the explorer one.

File: package.json [hunk 4]
```diff
@@ -3,33 +3,33 @@
...
-    "react": "^16.12.0",
-    "react-cookie-consent": "^6.2.4",
-    "react-copy-to-clipboard": "^5.0.1",
-    "react-dom": "^16.2.0",
-    "react-gtm-module": "^2.0.11",
-    "react-loading": "^1.0.3",
-    "react-paginate": "^5.0.0",
-    "react-redux": "^5.0.7",
+    "@hathor/wallet-lib": "1.1.0",
+    "axios": "0.17.1",
```
[the remark this answers is not in the document]
Comment (pedroferreira1) [reply package.json:7]: Yeah, I think this should be the top 1 priority of the upgrade dependencies project.

**Review by tuliomir — APPROVED** [review]

**Review by r4mmer — APPROVED** [review]
